ci/stress_tests/run_application_stress_tests.sh

```diff
@@ -85,10 +85,11 @@ test_impala(){
             RLLIB_DIR=../../python/ray/rllib/
             ray --logging-level=DEBUG up -y "$CLUSTER" &&
             ray rsync_up "$CLUSTER" $RLLIB_DIR/tuned_examples/ tuned_examples/ &&
-            sleep 1 &&
-            ray --logging-level=DEBUG exec "$CLUSTER" "rllib || true" &&
-            ray --logging-level=DEBUG exec "$CLUSTER" "
-                rllib train -f tuned_examples/atari-impala-large.yaml --ray-address='localhost:6379' --queue-trials" &&
+            # HACK: the test will deadlock if it scales up slowly, so we have to wait
```

Comment:
Any idea why we didn't run into this before?

Reply:
I think we did, but just retried until it got lucky.